src/validators/typed_dict.rs

```diff
@@ -119,6 +127,47 @@ impl BuildValidator for TypedDictValidator {
                 }
                 None => LookupKey::from_string(py, field_name),
             };
+
+            let required = match field_info.get_as::<bool>(intern!(py, "required"))? {
+                Some(required) => {
+                    if required && (default.is_some() || default_factory.is_some()) {
+                        return py_error!("Field \"{}\": a required field cannot have a default value", field_name);
```

Comment:
```suggestion
                        return py_error!("Field '{}': a required field cannot have a default value", field_name);
```
As a convention I mostly used single quotes inside errors to identify variables, it's cleaner and more like python's behaviour I think.